run.cmd

```diff
@@ -1,19 +1,24 @@
 @if "%_echo%" neq "on" echo off
 setlocal
 
-if defined VisualStudioVersion goto :Run
+if defined VisualStudioVersion (
+    if not defined __VSVersion echo %__MsgPrefix%Detected Visual Studio %VisualStudioVersion% developer command ^prompt environment
```

Comment:
I don't think this applies to this repo.

Reply:
will remove it